Dockerfile

```
@@ -15,11 +15,9 @@ COPY . .
 RUN make build-ethermint-linux
 
 # Final image
-FROM alpine:edge
+FROM golang:1.13 as final
```

should match the one on go.mod I think

```suggestion
FROM golang:1.14 as final
```